flex-checks/src/main/java/org/sonar/flex/checks/UnusedFunctionParametersCheck.java

```diff
@@ -140,11 +142,10 @@ private void reportUnusedArgument() {
     }
 
     if (nbUnusedArgs > 0) {
-      addIssue(
-        MessageFormat.format(
-          "Remove the unused function {0} \"{1}\".", nbUnusedArgs > 1 ? "parameters" : "parameter",
-          StringUtils.join(builder.toString().trim().split(" "), ", ")),
-        currentScope.functionDec);
+      String parameter = nbUnusedArgs > 1 ? "parameters" : "parameter";
+      String parameters = Pattern.compile(" ").splitAsStream(builder.toString().trim()).collect(joining(", "));
```

Comment:
I don't understand why we don't build desired format directly in the loop on line 139? It seems like building the string and splitting is entirely unnecessary.

Reply:
Changed